services/sleeper/src/sleeper/main.py

```diff
@@ -0,0 +1,110 @@
+import os
+import json
+import random
+import time
+import subprocess
+
+
+def get_from_environ(key, default=None):
+    """Returns a value from the environ if not found None"""
+    return os.environ.get(key, default)
```

Comment:
MINOR: IMO it does not bring much this helper function ...  or can use [``os.getenv(key, default)``](https://docs.python.org/3/library/os.html#os.getenv)